manifests/agent/logging.pp

```diff
@@ -0,0 +1,80 @@
+class classroom::agent::logging {
+
+  # Configure Graphite & Grafana
+  include '::apache'
+
+  file { '/opt/graphite':
+    ensure => 'directory',
+  }
+
+  apache::vhost { $::fqdn:
+    servername      => $::fqdn,
+    port            => 8081,
+    docroot         => '/opt/grafana',
+    error_log_file  => 'grafana_error.log',
+    access_log_file => 'grafana_access.log',
+    directories     => [
+      {
+        path            => '/opt/grafana',
+        options         => [ 'None' ],
+        allow           => 'from All',
+        allow_override  => [ 'None' ],
+        order           => 'Allow,Deny',
+      }
+    ]
+  }->
```

Comment:
can you turn these into `before => Class['grafana']` just in case students do end up looking at it? We teach that as best practice and try to maintain it when convenient.